core/tests/test_utils.py

```diff
@@ -342,14 +344,18 @@ def swap_is_feature_flag_enabled_function(
         context. The context with function replaced.
     """
     def mock_is_feature_flag_enabled(
```

Comment:
@lkbhitesh07 this probably needs fixing as well.

Reply:
Yep! I have added this part to the [issue](https://github.com/oppia/oppia/issues/19848) as well.
Thanks